epsagon/wrappers/python_function.py

```diff
@@ -67,12 +68,41 @@ def wrap_python_function(func, args, kwargs):
             pass
 
 
-def python_wrapper(func):
-    """Epsagon's general python wrapper."""
+def python_wrapper(*args, **kwargs):
+    """
+    Epsagon's general Python wrapper.
+
+    Receives optional keyword arg 'name': used to identify this resource in
+        the application (defaults to function name).
+
+    Options for using:
+    -   @python_wrapper(name='my-resource')
+        def my_function(params):
+            ...
+
+    -   @python_wrapper()
+        def my_function(params):
+            ...
+
+    -   @python_wrapper
+        def my_function(params):
+            ...
+
+    NOTE: Should not be used for two functions in the same stack trace, or when
+          running under a traced context (e.g. AWS Lambda, HTTP frameworks, etc)
```

Comment:
What happens if it IS used inside already traced code? (adding some noise to the graph in Epsagon is ok, as long as it has no implications on the instrumented code)

Reply:
The runner will be overridden - though other previous events will remain.

Not so good, but will not break.